utils/docker/run-build.sh

```diff
@@ -201,55 +201,55 @@ cmake .. -DCMAKE_BUILD_TYPE=Release \
 make -j$(nproc)
 #ctest --output-on-failure
 
-#if [ "$PACKAGE_MANAGER" = "" ]; then
+if [ "$PACKAGE_MANAGER" = "" ]; then
 	# install the library from sources
-#	sudo_password -S make -j$(nproc) install
-#else
+	sudo_password -S make -j$(nproc) install
+else
 	# Do not install the library from sources here,
 	# because it will be installed from the packages below.
 
-#	echo "##############################################################"
-#	echo "### Making and testing packages (RELEASE version) ..."
-#	echo "##############################################################"
+	echo "##############################################################"
+	echo "### Making and testing packages (RELEASE version) ..."
+	echo "##############################################################"
 
-#	make -j$(nproc) package
-#	find . -iname "libuasync*.$PACKAGE_MANAGER"
-#fi
+	make -j$(nproc) package
+	find . -iname "libuasync*.$PACKAGE_MANAGER"
+fi
 
-#if [ $PACKAGE_MANAGER = "deb" ]; then
-#	echo "$ dpkg-deb --info ./libuasync*.deb"
-#	dpkg-deb --info ./libuasync*.deb
+if [ $PACKAGE_MANAGER = "deb" ]; then
+	echo "$ dpkg-deb --info ./libuasync*.deb"
+	dpkg-deb --info ./libuasync*.deb
 
-#	echo "$ dpkg-deb -c ./libuasync*.deb"
-#	dpkg-deb -c ./libuasync*.deb
+	echo "$ dpkg-deb -c ./libuasync*.deb"
+	dpkg-deb -c ./libuasync*.deb
 
-#	echo "$ sudo -S dpkg -i ./libuasync*.deb"
-#	echo $USERPASS | sudo -S dpkg -i ./libuasync*.deb
+	echo "$ sudo -S dpkg -i ./libuasync*.deb"
+	echo $USERPASS | sudo -S dpkg -i ./libuasync*.deb
 
-#elif [ $PACKAGE_MANAGER = "rpm" ]; then
-#	echo "$ rpm -q --info ./libuasync*.rpm"
-#	rpm -q --info ./libuasync*.rpm && true
+elif [ $PACKAGE_MANAGER = "rpm" ]; then
+	echo "$ rpm -q --info ./libuasync*.rpm"
+	rpm -q --info ./libuasync*.rpm && true
 
-#	echo "$ rpm -q --list ./libuasync*.rpm"
-#	rpm -q --list ./libuasync*.rpm && true
+	echo "$ rpm -q --list ./libuasync*.rpm"
+	rpm -q --list ./libuasync*.rpm && true
 
-#	echo "$ sudo -S rpm -ivh --force *.rpm"
-#	echo $USERPASS | sudo -S rpm -ivh --force *.rpm
-#fi
+	echo "$ sudo -S rpm -ivh --force *.rpm"
+	echo $USERPASS | sudo -S rpm -ivh --force *.rpm
+fi
 
 #test_compile_all_examples_standalone
 
-#if [ "$PACKAGE_MANAGER" = "" ]; then
+if [ "$PACKAGE_MANAGER" = "" ]; then
 	# uninstall the library, since it was installed from sources
-#	cd $WORKDIR/build
-#	sudo_password -S make uninstall
-#elif [ $PACKAGE_MANAGER = "deb" ]; then
-#	echo "sudo -S dpkg --remove libuasync-dev"
-#	echo $USERPASS | sudo -S dpkg --remove libuasync-dev
-#elif [ $PACKAGE_MANAGER = "rpm" ]; then
-#	echo "$ sudo -S rpm --erase libuasync-devel"
-#	echo $USERPASS | sudo -S rpm --erase libuasync-devel
-#fi
+	cd $WORKDIR/build
+	sudo_password -S make uninstall
```

Comment:
I wonder if, since `make uninstall` is used here, shouldn't it, in the main `CMakeLists.txt` file, be added as a custom target.